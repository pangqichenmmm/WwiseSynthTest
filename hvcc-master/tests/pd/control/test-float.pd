#N canvas 365 398 347 238 10;
#X declare -path abs;
#X obj 34 9 loadbang;
#X obj 34 196 print;
#X obj 34 106 f 1;
#X msg 69 106 0.5;
#X obj 69 128 float;
#X msg 34 39 5;
#X floatatom 119 109 5 0 0 0 - - -, f 5;
#X obj 155 105 t b f b;
#X obj 155 144 f -34;
#X obj 101 9 declare -path abs;
#X obj 198 89 abs_float 3.33;
#X obj 34 66 t b b f f b;
#X connect 0 0 5 0;
#X connect 2 0 1 0;
#X connect 3 0 4 0;
#X connect 4 0 1 0;
#X connect 5 0 11 0;
#X connect 6 0 1 0;
#X connect 7 0 8 0;
#X connect 7 1 8 1;
#X connect 7 2 8 0;
#X connect 8 0 1 0;
#X connect 10 0 1 0;
#X connect 11 0 2 0;
#X connect 11 1 3 0;
#X connect 11 2 6 0;
#X connect 11 3 7 0;
#X connect 11 4 10 0;
