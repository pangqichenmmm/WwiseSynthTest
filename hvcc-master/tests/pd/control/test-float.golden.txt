[@ 0.000] print: 3.33
[@ 0.000] print: -34
[@ 0.000] print: 5
[@ 0.000] print: 5
[@ 0.000] print: 0.5
[@ 0.000] print: 1
